### PR TITLE
add section Mixing QuarkusTest and QuarkusIntegrationTest in getting-started-testing.adoc

### PR DESCRIPTION
**Pull request by sku0x20**
- update docs in ref to 
- https://github.com/quarkusio/quarkus/issues/43796 
- https://github.com/quarkusio/quarkus/issues/43804

@geoand

### CODE REVIEW

**Comment by geoand** [comment]
Thanks!

I would personally just have added something like this:

[NOTE]
----
Currently `@QuarkusTest` and `@QuarkusIntegrationTest` should not be run in the same test run.  

For Maven, this means that the former should be run by the surefire plugin while the latter should be run by the failsafe plugin.

For Gradle, this means the two types of tests should belong to different source sets.
----

WDYT?

**Comment by sku0x20** [comment]
i like this clear and concise. will update.  
my only concern is that users shouldn't be burdened with `how to add soucesets in gradle`, guide should provide a hint or a sample. i know its a trivial task, but still.

**Comment by geoand** [comment]
Then I guess after the gradle part, we can have a small example

File: docs/src/main/asciidoc/getting-started-testing.adoc [hunk 1]
```diff
@@ -1475,6 +1475,86 @@ You can use this tag to isolate the `@QuarkusTest` test in a specific execution
...
+====
+Currently `@QuarkusTest` and `@QuarkusIntegrationTest` should not be run in the same test run.  
+
+For Maven, this means that the former should be run by the surefire plugin while the latter should be run by the failsafe plugin.
+
+For Gradle, this means the two types of tests should belong to different source sets.
+====
+.gradle custom sourceset eg.
+[%collapsible]
+====
```
Comment (geoand) [inline docs/src/main/asciidoc/getting-started-testing.adoc:1488]: Shouldn't the snipper be inside the note?

Comment (sku0x20) [reply docs/src/main/asciidoc/getting-started-testing.adoc:1488]:
```
[NOTE]
====
note
[%collapsible]
====
inside
====
====
```
this was not rendering properly on https://asciidoc.org/ thats why i put it outside.

Comment (geoand) [reply docs/src/main/asciidoc/getting-started-testing.adoc:1488]: Something like https://stackoverflow.com/a/50823475/2504224 should work

Comment (sku0x20) [reply docs/src/main/asciidoc/getting-started-testing.adoc:1488]: thanks. updated.  
i haven't cloned this repo locally. can i see the rendered  version? how does the ci work?

Comment (geoand) [reply docs/src/main/asciidoc/getting-started-testing.adoc:1488]: See: https://quarkus-pr-main-43873-preview.surge.sh/version/main/guides/getting-started-testing#mixing-quarkustest-with-other-type-of-tests

**Comment by geoand** [comment]
Last thing, can you please squash the commits and also improve the `gradle custom sourceset eg.` title?

Thanks

**Comment by geoand** [comment]
I force pushed to your PR.

**Review by geoand — APPROVED** [review]

**Comment by geoand** [comment]
Thanks for the contribution!

**Comment by sku0x20** [comment]
thanks for the guidance.  :)
hope to contribute more.

**Comment by geoand** [comment]
🎉

**Pull request merged**